pkgs/development/python-modules/python-ndn/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pythonOlder
+, setuptools
+, pycryptodomex
+, lark
+, pygtrie
+, aenum
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-ndn";
+  version = "0.3-1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-+kWhUNu65yQMHP4m/86j0XqrZGrvtkYOLeLKXjzRhpY=";
+  };
+
+  disabled = pythonOlder "3.9";
+
+  propagatedBuildInputs = [
+    pycryptodomex
+    lark
+    pygtrie
+    aenum
+    setuptools
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportChecks = [ "ndn" ];
+
+  meta = with lib; {
+    description = "An NDN client library with AsyncIO support in Python 3";
+    homepage = "https://github.com/named-data/python-ndn";
+    changelog = "https://github.com/named-data/python-ndn/blob/master/CHANGELOG.rst#03-1-2022-03-20";
```

My guess is that `nix-update` wouldn't be able to add the changelog to the commit message.

```suggestion
    changelog = "https://github.com/named-data/python-ndn/blob/v${version}/CHANGELOG.rst";
```